arch/arm/core/Kconfig

```diff
@@ -34,6 +34,14 @@ config CPU_HAS_SYSTICK
 	help
 	  This option is enabled when the CPU has systick timer implemented.
 
+config BUILTIN_STACK_GUARD
```

Comment:
Should this be mutually exclusive with ARM_STACK_PROTECTION?

Reply:
@galak Definitely. I had planned to add this in the next PR, but, since you caught it, I add it here, now :)